gitsync: Реализация #112 через BDD

artbear opened the pull request:
Пример реализации конкретной задачи #112 через BDD-фичи с помощью проекта https://github.com/artbear/1bdd

Пулл-реквест создан для код-ревью и как пример реализации через BDD для 1script

PS В ветку develop изменения уже залиты.

artbear committed:
Заготовка фичи gitsync init и реализация первых шагов #112

artbear committed:
Добавил команды тестирования проекта для VSC

artbear committed:
Полность реализованы все шаги фичи для gitsync init для #112
Реализовано как правильное поведение, так и неправильное

artbear committed:
При инициализации через gitsync подкаталога, который находится внутри git-репозитория, не создается лишний каталог .git в этом подкаталоге fix #112

artbear committed:
Merge branch 'feature/gitsync-112' into develop

review-ninja commented:
<a href="https://app.review.ninja/EvilBeaver/oscript-library/pull/140" target="_blank"><img src="https://app.review.ninja/32417757/pull/140/badge" alt="ReviewNinja"/>

artbear committed:
Метод "ИмеетДлину" научился работать со всеми классами, у которых реализован метод "Количество"

artbear committed:
Слегка поправил текст фич для большей универсальности #112

EvilBeaver committed:
Обход ошибки 314 в OneScript

EvilBeaver committed:
Merge branch 'develop' of https://github.com/EvilBeaver/oscript-library into develop

EvilBeaver committed:
удалил лишние сообщения из теста

Commit:
Добавил Jenkinsfile для запуска тестирования в jenkis multibranch pipline

vitaliy-kotov committed:
Опечатка в тексте

artbear committed:
Merge pull request #143 from vitaliy-kotov/patch-1

Опечатка в тексте

artbear committed:
Merge pull request #142 from pumbaEO/feature/jenkinsfile

Добавил Jenkinsfile для запуска тестирования в jenkins multibranch pipeline

Commit:
Добавил установку локализации программы и сеанса

В контекст запуска добавил установку параметров языка локализации программы и языка локализации сеанса.  Параметры отвечающие за /L и /VL

EvilBeaver committed:
Merge pull request #144 from pumbaEO/feature/localeadd

Добавил установку локализации программы и сеанса

EvilBeaver commented:
И чего делать с пулреквестом, если изменения уже залиты?

pumbaEO commented:
Имеешь ввиду другие изменения? Или это уже в develop находится?

pumbaEO commented:
Понял вопрос: Тогда просто отменяешь pull request и все, внизу кнопка должна быть Close and comment.

artbear committed:
Обход проблемы OneScript #315
Если скрипт не является стартовым, вместо завершения работы системы использую выброс исключения

artbear committed:
Merge branch 'develop' of https://github.com/EvilBeaver/oscript-library into develop
